components/Form/index.js

```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components";
+
+export default function FormComponent({
+  onSubmitEvent = () => {}
+}) {
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+  
+    onSubmitEvent({
+      destination: data.destination,
+      from: data.from,
+      to: data.to,
+      entryTitle: data.entryTitle,
+      entry: data.entry,
+    });
+    event.target.reset();
+  }
+  return ( 
+<>
+  <StyledForm   onSubmit={(event) => handleSubmit(event)}>
+    <label htmlFor="destination">Your Destination:</label>
+    <StyledInput
+      name="destination"
+      pattern="^[^\sa0-9].*$"
+      required
+      id="1"
+      type="text"
+    />
```

Comment:
htmlFor has to match the id of the input, not the name. You need to change that for all of your label/input combinations

Reply:
changed it